### ISSUE TITLE
Install issue

### ISSUE THREAD

**Issue by kmsraidboss** [LordFetznschaedl/CS2Retake#5]
I know you mentioned not a lot of support right now but because the Install instructions are so basic...are there any common reasons this plugin won't load? Does it require a specific game_mode/type? 

What I've Tried: Running Linux server. I dropped into addons. Metamod is working (v1266), Counter Strike Sharp (build 65) is working and shows as loaded in `meta list`. I have tried running in Casual game mode (type 0, mode 0), Custom (3, 0), and Retakes (5, 0, 12) - all of which just load their default game mode. Ran on de_dust2 which has a spawn file. Plugin seems to be not loading as no Retakes commands work (despite having perms), spawns are default map spawns, and the Retakes config isn't loading. 

If you could possibly offer some advice, I'd be very grateful. I can't be the only one having an issue. Thank you so much for your help and for your work on this plugin.

**Comment by t1ckrate** [comment]
@kmsraidboss Hi, have you any logs or messages from the plugin ? Have you correctly placed the plugin in `counterstrikesharp/plugins` ? Is the plugin listed with the command `css_plugins list` ? 
The server needs to run competitive gamemode (not sure about it but works in my case) and `retake.cfg` configuration file needs to be placed in `csgo/cfg/cs2retake/` folder, you can retrieve it here [https://github.com/LordFetznschaedl/CS2-Configs/blob/master/retake/retake.cfg](https://github.com/LordFetznschaedl/CS2-Configs/blob/master/retake/retake.cfg)

**Comment by kmsraidboss** [comment]
Nothing about the plugin in Metamod logs. Plugin is correctly placed in counterstrikesharp/plugins, just as it is in the release. Nothing comes up when I use css_plugins list in console, but CounterStrikeSharp shows as running in `meta list`. `retake.cfg` is in csgo/cfg/cs2retake like it is in the release package. 

Everything seems right. I'm gonna try the competitive game mode part but I don't think that's it.

**Comment by t1ckrate** [comment]
@kmsraidboss Yes, the plugin has some issues when running into another gamemode than Competitive. Let us know if the changes made the plugin works.

**Comment by kmsraidboss** [comment]
I have resolved the issue. Will explain below plus leave a couple other notes regarding other things mentioned in this thread

- It was actually a CSS issue from the start. `meta list` showed CounterStrikeSharp running but when I typed `css_plugins list`, it would say not a valid command or whatever. That was on a very new version of CSS. I loaded an older version that my other servers use currently and once I did, `css_plugins list` worked and showed Retakes as a loaded plugin. 

- However, ironically, Retakes would get stuck on team select screen when running Competitive game mode (game_type 0, game_mode 1), which is what you mentioned it should be set as. It would show like 2 days worth of time on the timer up top and never let you choose a team or go to spec, and it would never put you on a team when 2+ players were connected/queueing. This happens with 1 player on the server as well as 2 or more. 

When running casual game mode (game_type 0, game_mode 0), the team select screen showed a 10 second countdown followed by automatically joining players to a team. Then the game mode functioned mostly proper. Gonna leave a couple bug reports below for whatever they are worth to the developers here:


[BUGS]

- Text in middle sometimes shows pistol round, but equips SMG or greater. 
- Bomb doesn't always spawn in hands, you sometimes have to toggle to it (press 5 or E)
- If you don't plant the bomb within the 6 seconds, the round doesn't end, it continues for almost 5 minutes (or until a team dies)
- If you don't plant the bomb within the time limit, then plant it AFTER chat says "Bomb wasn't planted in time", the bomb planter will get stuck in their spawn/plant position.


Thank you for your help @t1ckrate , I'm sorry for the trouble, especially since it ended up being CSS not fully loading or something, but your responses helped me get to that answer, so I appreciate it greatly.

**Comment by t1ckrate** [comment]
Great to hear that you have made it work ! @kmsraidboss 
I've encounter the same issue with a older version of CSS than CSRetake used for the compilation, that made the plugin to not load or encounter few errors. 
@LordFetznschaedl : It would be great if you can write the version of CSS with the one you have compiled the plugin in the Release notes or README.md. Few times that i wasn't sure if i need to update CSS to make it work with CSRetake.

Few fixes as been made on the queue system, that can explain the problem you had. I've haven't tested it yet but may be the issue you were running into. We've heard a lot of errors if the server isn't running in competitive gamemode, but if you made it work on casual, it's good to know. And thanks for the bugs list ! 
Best.

**Comment by LordFetznschaedl** [comment]
> @LordFetznschaedl : It would be great if you can write the version of CSS with the one you have compiled the plugin in the Release notes or README.md. Few times that i wasn't sure if i need to update CSS to make it work with CSRetake.

Will be done. The required CSS version currently is 66.

**Comment by LordFetznschaedl** [comment]
Thank you @t1ckrate for helping.

**Comment by kmsraidboss** [comment]
@LordFetznschaedl 

- My original posts were based on CS2Retake v1.0.0 w/ CSS Build 65. So anything I wrote prior to this message was on that setup. 

- I just now updated to the latest CS2Retake v1.0.2 Alpha w/ CSS Build 66. I only tested with myself and 1 other person but the T's do seem to die automatically now if the bomb is not planted in time. 

- This is just a thought regarding the Bomb Plant situation. The bomb doesn't spawn in hand until the 3 second freezetime ends. Then it gives "5 seconds" to plant. I use quotes on "5 seconds" because it feels like it's not that long. A player has to tap E once or twice AFTER freezetime to get the bomb in hand. 

So I was thinking that maybe upping that to 7-8 seconds until some auto-plant thing is worked out would be nice so players have time to react. 

- Lastly, and I didn't notice this but my testing friend mentioned it so I'm just gonna say something, he thought that maybe there was an issue with crouch jumping too high. Idk if it was a server thing or plugin thing, didn't notice myself. 

Thanks for all the support, I'll keep updating as I run into things <3
